packages/lesswrong/server/migrations/20221213T162530.fix_broken_sequences.ts

```diff
@@ -0,0 +1,36 @@
+import Chapters from "../../lib/collections/chapters/collection";
+import Sequences from "../../lib/collections/sequences/collection";
+import { randomId } from "../../lib/random";
+import InsertQuery from "../../lib/sql/InsertQuery";
+
+export const up = async ({db}: MigrationContext) => {
```

Comment:
This is, like, a data-quality migration not a schema migration. I thought we were doing to do something different for data quality migrations.

Reply:
I don't think we really discussed that (that I can remember), and I don't think there's any reason to separate them in the case that it only needs to be run once (like here). If it needed to be run regularly then I would've made it a script instead of a migration though.

This was also created with `yarn migrate create --name=fix_broken_sequences` rather than using any of the `makemigrations` stuff.